components/form/builder/README.md

```diff
@@ -33,15 +33,49 @@ Defines the form items and dependencies between fields.
     type: 'select',
     label: 'Countries',
     placeholder: 'Select a country',
-    errorText: 'Required',
-    next: 'city'
+    next: 'city',
+    empty: {
+      text: 'Required'
+    }
   },
   city: {
     type: 'select',
     label: 'Cities',
     placeholder: 'Select a city',
-    errorText: 'Required',
-    next: 'street'
+    next: 'street',
+    empty: {
+      text: 'Required'
+    }
+  },
+  comments: {
+    type: 'text-area',
+    label: 'Comments',
+    placeholder: 'Type your comments'
+    persists: true,
+    errors: {
+      empty: {
+        text: 'Required'
+      },
+      notAllowed: {
+        character: {
+          text: 'Do not enter special characters',
+          condition: /[`~@#$^&¬|=<>{}[\]\\]/
```

Comment:
I prefer "pattern" over "condicion" following W3C https://www.w3.org/TR/2011/WD-html5-20110525/common-input-element-attributes.html#the-pattern-attribute

Reply:
Not mandatory, just to think about

Reply:
DONE!